js-api-spec/function.test.ts

```diff
@@ -226,14 +101,29 @@ describe('gracefully handles a custom function', () => {
     ).toThrowSassException({line: 0});
   });
 
-  it('returning a non-Value', () => {
-    expect(() =>
-      compileString('a {b: foo()}', {
-        functions: {
-          'foo()': (() => 'wrong') as unknown as CustomFunction<'sync'>,
-        },
-      })
-    ).toThrowSassException({line: 0});
+  describe('returning a non-Value', () => {
+    it('before simplification', () => {
```

Comment:
"Before" and "after" simplification is a very implementation-focused way of describing this. I'd make it more focused on the caller's perspective, and say "directly" versus "in a calculation"